packages/firestore/src/remote/persistent_stream.ts

```diff
@@ -677,7 +677,7 @@ export class PersistentWriteStream extends PersistentStream<
    * PersistentWriteStream manages propagating this value from responses to the
    * next request.
    */
-  lastStreamToken: ProtoByteString;
+  lastStreamToken: ProtoByteString = '';
```

Comment:
Can we use `emptyByteString()` here? https://github.com/firebase/firebase-js-sdk/blob/df1b58816a5f502853f4b6fde59033abca92f3c9/packages/firestore/src/platform/platform.ts#L85

Reply:
Yes, good call!